sale_financial_risk/models/sale.py

```diff
@@ -33,21 +34,22 @@ def evaluate_risk_message(self, partner):
 
     def action_confirm(self):
         if not self.env.context.get("bypass_risk", False):
-            partner = self.partner_invoice_id.commercial_partner_id
-            exception_msg = self.evaluate_risk_message(partner)
-            if exception_msg:
-                return (
-                    self.env["partner.risk.exceeded.wiz"]
-                    .create(
-                        {
-                            "exception_msg": exception_msg,
-                            "partner_id": partner.id,
-                            "origin_reference": "{},{}".format("sale.order", self.id),
-                            "continue_method": "action_confirm",
-                        }
+            for order in self:
+                partner = self.partner_invoice_id.commercial_partner_id
```

Comment:
Wondering if we should take it from `order` instead of `self`?
```suggestion
                partner = order.partner_invoice_id.commercial_partner_id
```

Reply:
Please change this, @rousseldenis